mobile/lib/ui/settings/backup/backup_item_card.dart

```diff
@@ -137,17 +141,36 @@ class _BackupItemCardState extends State<BackupItemCard> {
                     color: getEnteColorScheme(context).fillBase,
                   ),
                   onPressed: () {
-                    String errorMessage = "";
-                    if (widget.item.error is Exception) {
-                      final Exception ex = widget.item.error as Exception;
-                      errorMessage = "Error: " +
-                          ex.runtimeType.toString() +
-                          " - " +
-                          ex.toString();
-                    } else if (widget.item.error != null) {
-                      errorMessage = widget.item.error.toString();
-                    }
-                    showErrorDialog(context, 'Upload failed', errorMessage);
+                    showDialogWidget(
```

Comment:
Can you log the here as warning, so that whosoever is looking at log for such logs can immediately see what went wrong.

Reply:
Done! I have logged it in "Contact support" button click.